src/libaktualizr/primary/aktualizr.cc

```diff
@@ -1,10 +1,11 @@
-#include "aktualizr.h"
-
-#include <chrono>
-
 #include <sodium.h>
+#include <chrono>
```

Comment:
Not a big deal, but C++ headers should be listed before external libs.

Reply:
It's not my fault,  ```make format``` makes this changed(:

Reply:
Even if you put empty lines between the C++ and external header sections?

Reply:
thanks it works